src/Lesson1/Task1/ArrayExcerciseVariantTwo.java

```diff
@@ -0,0 +1,53 @@
+package Lesson1.Task1;
+
+/*В массиве целых чисел определить сумму элементов, состоящих на
+чётных позициях.
+*/
+
+/**
+ * @author Andrii Prokofiev
+ * @version 1.0
+ */
+
+public class ArrayExcerciseVariantTwo {
+    public static void main(String[] args) {
+        ArrayExcerciseVariantTwo arrayExcerciseVariantTwo = new ArrayExcerciseVariantTwo();
+        arrayExcerciseVariantTwo.startProgram();
+    }
+
+    public void startProgram() {
+
+        Tools.printArray(getIntArray());
+        System.out.println(getSum(getIntArray()));
+    }
+
+    public static int getSum(int[] array) {
+        int sum = 0;
+        for (int i = 0; i < array.length; i++) {
+            if (i > 0 && i % 2 == 0) {
+                sum = sum + array[i];
+            }
+        }
+        return sum;
+    }
+
+
+    public static int[] getIntArray() {
```

Comment:
You have same method in previous task, so it's time to prepare some utils method to work with array to avoid code duplication.

Reply:
When you will implement common method please, think make it reusable. So think how to write it to generate not only values from -10 to 10, but any other values range.